ambari/README.md

```diff
@@ -23,27 +23,19 @@ start collecting your Ambari performance data. See the [sample ambari.d/conf.yam
 [Run the Agent's status subcommand][4] and look for `ambari` under the Checks section.
 
 
+## Data collected
 ### Metrics
 
-This integration collects for every host in every cluster the following system metrics:
-
-* boottime
-* cpu
-* disk
-* memory
-* load
-* network
-* process
-
-If service metrics collection is enabled with `collect_service_metrics` this integration will collect for each
-whitelisted service component the metrics with headers in the white list.
```

Comment:
If you want to keep this information, it can be put between `## Data Collected` and `### Metrics`